### PR TITLE
Bond balances in genesis

### PR DESCRIPTION
**Pull request by vicsn**
## Motivation

We want to be able to bond balances in the genesis block

## Related PRs

https://github.com/AleoHQ/snarkVM/pull/2308

### CODE REVIEW

File: node/router/src/lib.rs
Comment (d0cd) [inline node/router/src/lib.rs:1]: Are these changes relevant?

File: node/bft/tests/common/primary.rs [hunk 1]
```diff
@@ -351,24 +352,31 @@ fn genesis_block(
...
     // Initialize the genesis block.
-    vm.genesis_quorum(&genesis_private_key, committee, public_balances, rng).unwrap()
+    vm.genesis_quorum(&genesis_private_key, committee, public_balances, bonded_balances, rng).unwrap()
 }
 
 fn genesis_ledger(
     genesis_private_key: PrivateKey<CurrentNetwork>,
     committee: Committee<CurrentNetwork>,
     public_balances: IndexMap<Address<CurrentNetwork>, u64>,
+    bonded_balances: IndexMap<Address<CurrentNetwork>, u64>,
```
Comment (d0cd) [inline node/bft/tests/common/primary.rs]:
```suggestion
    bonded_balances: IndexMap<Address<CurrentNetwork>, (Address<CurrentNetwork>, u64)>,
```

Comment (d0cd) [reply node/bft/tests/common/primary.rs]: Ditto for the rest of type signatures

**Comment by d0cd** [comment]
Should this target the `mainnet` branch?

**Review by d0cd — DISMISSED** [review]

**Comment by raychu86** [comment]
Closing in favor of https://github.com/AleoHQ/snarkOS/pull/3091

**Pull request closed without merging**